contentprovider/README.md

```diff
@@ -17,7 +17,7 @@ Then, add the following dependencies:
 dependencies {
    ...
 
-   compile 'org.literacyapp:literacyapp-model:1.1.56'
+   compile 'org.literacyapp:literacyapp-model:1.1.67'
```

Comment:
I think you need to change this to "compile 'ai.elimu:model:1.1.67'" as shown under "Choose dependency snippet -> Gradle" https://bintray.com/elimu-ai/maven/model

Reply:
Yes, you are right. Thanks for reminding me :-)